.github/workflows/tav.yml

```diff
@@ -24,21 +24,34 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     outputs:
-      versions: ${{ steps.generate-matrix.outputs.versions }}
-      modules: ${{ steps.generate-matrix.outputs.modules }}
+      versions: ${{ steps.transform.outputs.versions }}
+      modules: ${{ steps.transform.outputs.modules }}
     steps:
-    - uses: actions/checkout@v3
-    ## TODO: use elastic/apm-pipeline-library/.github/actions/version-framework
-    ##       as soon as https://github.com/elastic/apm-pipeline-library/issues/2171 is done.
-    - id: generate-matrix
-      run: .ci/generate-github-tav-matrix.sh
+
+      - uses: actions/checkout@v3
+
+      - id: transform
+        name: Transform comment to the supported matrix
```

Comment:
nit: There is no comment in use for this workflow, so perhaps a name like "load matrix from tav.json" or something. However, I can understand if you want to keep the same `id` and `name` as the step in the "tav-command.yml" workflow.

Reply:
```suggestion
      - id: transform
        name: Load matrix from tav.json
```